### ISSUE TITLE
Why is the mIoU on the test set (67.8) higher than the mIoU on the validation set (65.9)?

### ISSUE THREAD

**Issue by jialeli1** [xinge008/Cylinder3D#76]
The performance on the test set is usually lower than the performance on the validation set.  Why is the mIoU on the test set (67.8) higher than the mIoU on the validation set (65.9)?

**Comment by xinge008** [comment]
The submission on leaderboard is trained with all labeled data (including Training and Validation set). Other techniques including TTA (test time augmentation) are also involved.

**Comment by jialeli1** [comment]
Thanks for your kind reply.
